src/wallet/wallet.ts

```diff
@@ -994,6 +1007,31 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
     const tx = await this.prepareDestroyAuthorityData(token, type, count);
     return this.handleSendPreparedTransaction(tx);
   }
+
+  /**
+   * Create an NFT in the network
+   *
+   * @memberof HathorWalletServiceWallet
+   * @inner
+   */
+  async createNFT(name: string, symbol: string, amount: number, data: string, options = {}): Promise<Transaction>  {
+    this.failIfWalletNotReady();
+    type optionsType = {
```

Comment:
What is this for?

Typescript will only perform static type checking at compile time, so I guess this is pointless unless you type the method parameter

Reply:
As we discussed in a call, I've been using this instead of typing the parameter to prevent using `| undefined` in all parameters because they are optional inside options. I understand it won't execute type checking on it but I feel it's better than not to have because anyone reading the code can clearly understand what are the types. I will create an issue to study further what would be the best approach for this situation and we can refactor all methods that are using this.